instance_clusters.go

```diff
@@ -37,6 +37,7 @@ type InstanceCluster struct {
 	ID                 int                 `json:"id"`
 	Name               string              `json:"name"`
 	Domain             string              `json:"domain"`
+	Managed            bool                `json:"managed"`
```

Comment:
@Starttoaster could you maybe check this one as well? I noticed the docs mention this as an option when adding a cluster (https://docs.gitlab.com/ee/api/instance_clusters.html#add-existing-instance-cluster) but not sure if it actually is a field in the response...

Reply:
Absolutely! This looks like a self-hosted feature, and I'm personally a user of GitLab.com, but to be honest I've been meaning to deploy my own instance one of these days anyway. I'll take a look and get back to this.

Reply:
Thanks! But I don't want to put a burden on your time, so please only have a look if it fits your personal schedule and interest!